.github/workflows/npm-cd.yml

```diff
@@ -220,6 +220,19 @@ jobs:
                     fi
                   done
 
+            - name: Check if RC and set tag for the package
+              shell: bash
+              run: |
+                  if [[ "${GITHUB_REF:11}" == *"rc"* ]]
+                  then
+                    echo "This is a release candidate"
+                    export tag="next"
+                  else
+                    echo "This is a stable release"
+                    export tag="latest"
+                  fi
+                  echo "TAG=${tag}" >> $GITHUB_ENV
```

Comment:
minor: since `tag` is being used also in a github terms, lets be clear - NPM_TAG, or DIST_TAG